draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -253,9 +253,11 @@ Each `K_i` MUST be constant in length, therefore the secret shares `SS_i` can be
 K_i = H(SS_i)
 ~~~
 
+Any protocols making use of this construction MUST either hash all inputs `SS_i`, or justify that any un-hashed inputs will always be fixed length.
+
 ## Protocol binding
 The `fixedInfo` string is a fixed-length string containing some context-specific information.
-It MUST NOT depend on the secret shares.
+It MUST NOT depend on the secret shares. The intention is to prevent cross-protocol attacks by making this key derivation unique to its protocol context.
```

Comment:
```suggestion
It MUST NOT depend on the secret shares. The intention is to prevent cross-context attacks by making this key derivation unique to its protocol context.
```

I would not use the cross-protocol term, as I would not expect data structures from two different protocols to be suitable. But definitely in the same protocol, but different contexts. If you have a better term feel free to suggest!